### PR TITLE
Add some questions.

### PR DESCRIPTION
**Pull request by tarekdj**
This PR introduces 2 new questions.

### CODE REVIEW

**Comment by ismail1432** [comment]
Thanks for your contribution, I'll take asap :+1:

File: src/Infrastructure/InMemory/InMemoryQuestionRepository.php [hunk 1]
```diff
@@ -135,6 +135,18 @@ class InMemoryQuestionRepository implements QuestionRepositoryInterface
             'choices' => ['Python', 'Ruby', 'Java', 'PHP', 'Swift'],
             'answer' => 4,
         ],
+        [
+            'id' => '43433d7c-1934-4294-bacd-b013aae65de7',
+            'question' => 'Github is an open source platform for hosting source code.',
+            'choices' => ['True', 'False'],
```
Comment (ismail1432) [inline src/Infrastructure/InMemory/InMemoryQuestionRepository.php:141]: I would answer true :sweat_smile:

File: tests/Repository/QuestionRepositoryTest.php [hunk 2]
```diff
@@ -141,6 +141,16 @@ private function getExpectedAnswers(): array
                 'choices' => ['Python', 'Ruby', 'Java', 'PHP', 'Swift'],
                 'answer' => 'Swift',
             ],
+            [
+                'question' => 'Github is an open source platform for hosting source code.',
```
Comment (ismail1432) [inline tests/Repository/QuestionRepositoryTest.php:145]: thanks for the test

**Comment by tarekdj** [comment]
GH is definitely NOT open source 😁

Le mer. 9 mars 2022 17:58, Smaine Milianni ***@***.***> a
écrit :

**Comment by ismail1432** [comment]
THANKS @tarekdj :+1: 

I will think of a way to quote you as an author on these questions.

**Pull request merged**